packages/web3-eth-accounts/src/index.js

```diff
@@ -235,7 +234,7 @@ Accounts.prototype.signTransaction = function signTransaction(tx, privateKey, ca
             var rawTransaction = '0x' + rlpEncoded;
             var transactionHash = utils.keccak256(rawTransaction);
 
-            return {
+            var result = {
```

Comment:
Not a big deal because of how `var` works (hoisting) but since `result` is referenced outside the try block that starts on L173 it might be more clear if `result` is declared outside of it with `var result;` and then the assignment is made with `result =` instead of `var result =`.

Reply:
Ah! It's no longer referenced anywhere outside the try block... believe the old references deleted just below this point were the only ones.  

Thanks for looking this over @michaelsbradleyjr!

Reply:
Ah, I see, I missed that aspect of the changes. Thanks for clarifying!